app/src/main/java/pm/gnosis/heimdall/data/repositories/impls/WalletConnectBridgeRepository.kt

```diff
@@ -243,6 +225,61 @@ class WalletConnectBridgeRepository @Inject constructor(
             config.handshakeTopic
         }
 
+    private fun handleCustomCall(sessionId: String, session: Session, call: Session.MethodCall.Custom) {
+        sessionRequests[call.id] = sessionId
+        try {
+            when (call.method) {
+                "gs_multi_send" -> {
```

Comment:
should we extract it to enum or some const?

Reply:
We could put it into a const, enum doesn't really make sense imo